glass/src/libnt/native/include/glass/networktables/NetworkTables.h

```diff
@@ -120,6 +122,7 @@ class NetworkTablesFlagsSettings {
   bool* m_pShowFlags = nullptr;
   bool* m_pShowTimestamp = nullptr;
   bool* m_pCreateNoncanonicalKeys = nullptr;
+  unsigned char m_precision;
```

Comment:
Changes aren't being saved because this isn't a pointer.  To save it, change to `int* m_pPrecision` and use `*m_pPrecision` everywhere, and use `&` when getting the storage for it.